plugins/nexus-repository-manager/src/utils/get-file-size/get-file-size.test.ts

```diff
@@ -14,8 +14,9 @@ describe('getFileSize', () => {
     };
     const rawAssets = [
       {
-        schemaVersion: 2,
-        mediaType: '',
+        schemaVersion: 2 as const,
```

Comment:
maybe the `it()` should get a new description text also

Reply:
I agree with @tplass-ias here, can you please update the testcase description here 👍

Reply:
done 👍